gulpfile.babel.ts

```diff
@@ -185,78 +162,72 @@ export function styles() {
     .pipe(connect.reload());
 }
 
-export function processJavascripts() {
+function processJavascript() {
   return gulp
-    .src(paths.javascripts.src, { since: gulp.lastRun(processJavascripts) })
+    .src(paths.javascript.src, { since: gulp.lastRun(processJavascript) })
     .pipe(
       babel({
         comments: false,
       }),
     )
     .pipe(gulpIf(!isDevBuild, terser())) // Only uglify in production to speed up dev builds
-    .pipe(gulp.dest(paths.javascripts.dest))
+    .pipe(gulp.dest(paths.javascript.dest))
     .pipe(connect.reload());
 }
 
-export function processTypescripts() {
+function processTypescript() {
   return gulp
-    .src(paths.typescripts.src, { since: gulp.lastRun(processTypescripts) })
+    .src(paths.typescript.src, { since: gulp.lastRun(processTypescript) })
     .pipe(tsProject())
     .js.pipe(
       babel({
         comments: false,
       }),
     )
     .pipe(gulpIf(!isDevBuild, terser())) // Only uglify in production to speed up dev builds
-    .pipe(gulp.dest(paths.typescripts.dest))
+    .pipe(gulp.dest(paths.typescript.dest))
     .pipe(connect.reload());
 }
 
-export function watch() {
+function watch() {
   gulp.watch(paths.styles.watch, styles);
 
   gulp.watch([paths.src], mvSrc);
 
-  gulp.watch(paths.javascripts.watch, processJavascripts);
-  gulp.watch(paths.typescripts.watch, processTypescripts);
+  gulp.watch(paths.javascript.watch, processJavascript);
+  gulp.watch(paths.typescript.watch, processTypescript);
 }
 
-export function webserver() {
+function webserver() {
   connect.server({
     root: paths.dest,
     livereload: true,
   });
 }
 
-export function recipes() {
+function recipes() {
   return gulp
     .src(paths.recipes.src, { since: gulp.lastRun(recipes) })
     .pipe(gulp.dest(paths.recipes.dest));
 }
 
-export function recipeInfo() {
```

Comment:
Removing all the other `export` - is that due to the reason that they are not invoked from other files anyways?

Reply:
yes, kind of. As far as i have checked, the gulp file should dev and build variable to be exported.
I have used `CLEAN=true ./scripts/build-unix.sh` to build it and it worked in local.